### PR TITLE
(Feature) - Orderbook: Pad sizes with zeros 

### LINKED ISSUE #28: Orderbook details
- [x] Parent of #42 (Orderbook is not subtracting filled values)
- [x] Parent of #59 (Add 'My Size' column to Orderbook)
- [ ] Parent of #60 (Pad sizes with 0)

*More items to be added*

### LINKED ISSUE #60: Orderbook: Pad sizes with zeroes

### PR DESCRIPTION
**Pull request by mariano-aguero**
Closes #60 
Also closes #28 

- Added function to calculate the decimals to add to a given number
- Added tests
- Added component to show numbers with different colors

Should look like this now
screenshot_20190226_111844

### CODE REVIEW

File: src/util/number_utils.ts [hunk 1]
```diff
@@ -0,0 +1,20 @@
...
+export const padRightSplitted = (numBg: BigNumber, decimals: number = 4): { num: string; diff: string } => {
+    const numBgToFixed = numBg.toFixed(decimals);
+    const numBgToString = numBg.toString();
+
+    const decimalPlaces = (numBgToString.split('.')[1] || []).length;
+
+    let diff = '';
+    let num = numBgToFixed;
+    if (!numBg.isZero() && decimalPlaces < decimals) {
+        diff = numBgToFixed.split(numBgToString).join('');
```
Comment (fvictorio) [inline src/util/number_utils.ts]: This is equivalent to `.replace(numBgToString, '')`, isn't it?

Comment (mariano-aguero) [reply src/util/number_utils.ts]: Yes @fvictorio , Do you think the replace is a better option?

**Comment by mariano-aguero** [comment]
> Minor comment, but otherwise looks good. The only thing pending is for the numbers to be tabular. I think that can be done adding this CSS:
> 
> ```
> font-feature-settings: 'tnum' 1
> ```
> That's probably something for @gabitoesmiapodo, but if that's the only thing needed here, maybe we can just add it.

@fvictorio Do you want to create another issue? or we keep this issue open until @gabitoesmiapodo  solve it here

**Comment by gabitoesmiapodo** [comment]
@mariano-aguero you have my blessing for this.

File: src/util/number_utils.ts (hunk 1 above)
Comment (fvictorio) [reply src/util/number_utils.ts]: Yes

**Comment by fvictorio** [comment]
> Do you want to create another issue? or we keep this issue open until @gabitoesmiapodo solve it here

Let's do it here.

File: src/util/number_utils.ts (hunk 1 above)
Comment (mariano-aguero) [reply src/util/number_utils.ts]: Solved

**Comment by gabitoesmiapodo** [comment]
@fvictorio @mariano-aguero 

This should be fixed now.

> Minor comment, but otherwise looks good. The only thing pending is for the numbers to be tabular. I think that can be done adding this CSS:

**Pull request merged**